stellar-auth/pkg/utils/password_validation.go

```diff
@@ -83,10 +111,28 @@ func ValidatePassword(input string) error {
 	if len(invalidCharacteres) > 0 {
 		failedValidations["invalid character"] = fmt.Sprintf("password cannot contain any invalid characters ('%s')", strings.Join(invalidCharacteres, "', '"))
 	}
+	if !hasLength {
+		failedValidations["length"] = fmt.Sprintf("password length must be between %d and %d characters", passwordMinLength, passwordMaxLength)
+	}
 
 	if len(failedValidations) == 0 {
-		return nil
+		if pv.determineIfCommonPassword(input) {
+			failedValidations["common password"] = "password is determined to be too common"
+		} else {
+			return nil
+		}
 	}
 
 	return &ValidatePasswordError{FailedValidationsMap: failedValidations}
 }
+
+func (pv *PasswordValidator) determineIfCommonPassword(input string) bool {
+	input = strings.ToLower(input)
```

Comment:
❓ Just checking, but are all passwords in `common-passwords.txt.gz` lowercased and have white spaces trimmed already ?

Reply:
yes, I already took this into account when generating the list